app/src/main/res/values-night/themes.xml

```diff
@@ -1,6 +1,6 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Detele this file - for now we don't need dark mode for this app - it will be easier to work with styles - when there are only one themes.xml